cogs/admin.py

```diff
@@ -0,0 +1,198 @@
+        """
+        Loads a cog.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            cog: The filename of the cog to be loaded.
+
+        Replies:
+            A success message.
+        """
+        self.bot.load_extension(f"cogs.{cog}")
+
+        embed = Embed(title="Cog", description=f"The {cog} cog was loaded successfully.")
+        await ctx.reply(embed=embed)
+
+    @module.group(pass_context=True)
+    async def reload(self, ctx: Context, cog: str):
+        """
+        Reloads a cog.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            cog: The filename of the cog to be reloaded.
+
+        Replies:
+            A success message.
+        """
+        self.bot.unload_extension(f"cogs.{cog}")
+        self.bot.load_extension(f"cogs.{cog}")
+        embed = Embed(title="Cog", description=f"The {cog} cog was reloaded successfully.")
+        await ctx.reply(embed=embed)
+
+    @module.group(pass_context=True)
+    async def unload(self, ctx: Context, cog: str):
+        """
+        Unloads a cog.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            cog: The filename of the cog to be unloaded.
+
+        Replies:
+            A success message.
+        """
+        self.bot.unload_extension(f"cogs.{cog}")
+        embed = Embed(title="Cog", description=f"The {cog} cog was unloaded successfully.")
+        await ctx.reply(embed=embed)
+
+    # Purge group
+
+    @group(pass_context=True)
+    async def purge(self, ctx: Context):
+        """
+        A group of commands.
+        """
+        if not ctx.invoked_subcommand:
+            raise BadArgument
+        member: Union[Member, User] = ctx.author
+        logger.info(f'User="{member.name}#{member.discriminator}({member.id})", Command="{ctx.message.content}"')
+
+    @purge.group()
+    @is_owner()
+    async def chat(self, ctx: Context, count: int):
+        """
+        Cleans a chat.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            count: The amount of messages to be cleaned.
+
+        Replies:
+            A success message containing the deleted amount of messages.
+        """
+        count = len(await ctx.channel.purge(limit=count, bulk=True))
+        embed = Embed(title=ctx.channel.name,
+                      description=f"Purged {count} messages")
+        await ctx.send(embed=embed)
+
+    @purge.group()
+    @commands.bot_has_guild_permissions(administrator=True)
+    async def member(self, ctx: Context, after, guild_purge: bool = False):
+        """
+        Cleans a chat.
+
+        Args:
+            ctx: The command context provided by the discord.py wrapper.
+
+            after: Messages younger than after will be deleted.
+
+            guild_purge: Determines if only one chat shall be purged or all chats contained by the guild.
```

Comment:
may not be a good idea to have this option to purge a whole Guild, mistakes a bound to happen. Remove the option

Reply:
Thought it could be useful to wipe out messages from a troll or something.
But okay, we can remove this option.